mlir/lib/Bindings/Python/IRCore.cpp

```diff
@@ -450,6 +458,9 @@ class PyBlockList {
 
   PyBlock dunderGetItem(intptr_t index) {
     operation->checkValid();
+    if (index < 0) {
+      index += dunderLen();
+    }
```

Comment:
```suggestion
    if (index < 0)
      index += dunderLen();
```

Nit: spurious braces

Reply:
I see what you are saying, but lots of other one-line conditionals in this file are written in this way.

Reply:
Sure, happy to do that as a separate PR. Resolving this and other comments for now.